src/ModalPreviewLayout/ModalPreviewLayout.st.css

```
@@ -11,10 +11,10 @@
 }
 
 .overlay {
-  align-self: start;
+  /* align-self: start; */
   width: 100vw;
   height: 100vh;
-  overflow-y: auto;
+  overflow-y: hidden;
   overflow-x: hidden;
```

If you hide both `overflow-x` & `overflow-y` it could be just:
```suggestion
  overflow: hidden;
```